docs/doxygen/overviews/xrc_format.h

```diff
@@ -2733,6 +2733,9 @@ support the following properties:
     optionally the proportion can be appended after each number
     separated by a @c :
     (default: none).}
+@row3col{empty_cellsize, @ref overview_xrcformat_type_size,
+    Size used for cells in the grid with no item. (default: @c wxDefaultSize).
```

Comment:
I wonder if it could be useful to link to `wxGridBagSizer::SetEmptyCellSize()` here.

Reply:
Yes, I guess I mostly thought about doing it because I wasn't sure myself what this was, but it's true that I wouldn't have found much from our current docs... Let's leave it like this then and hopefully remember to add a link here if the documentation of the function itself is ever improved.